src/main/webapp/app/shared/util/firebase/firebase-crud-store.ts

```diff
@@ -70,14 +71,23 @@ export class FirebaseCrudStore<T> {
     }
   }
 
+  async updateUntemplated(path: string, value: any) {
```

Comment:
What does this name mean?

Reply:
The update function currently used is templated depending on the store. For example, if we have `FireBaseCrudStore<Gene>`, then the update function looks like `update(path: string, value: Gene)`. I think it is useful to have an untemplated version so we can write anywhere without requiring a store.